### DISCUSSION TITLE
Vídeo inicial da apresentação

### DISCUSSION THREAD

**Discussion by gabrielsousarod** [EasyRoomCompany/Guardian#59] [Ideas]
amanhã vou gravar e editar o vídeo inicial da apresentação, a ideia é mostrar alguém precisando de uma sala e entrando no nosso site e mostrar alguns takes de sala de reuniões. o que acham?

**Comment by ivannaprata** [comment]
Boa ideia. Quando fizer, avisa para que a equipe possa assistir.

**Comment by kevinDsousa** [comment]
Certo, precisamos @fabianopmelo @joellacerda @enzogomezz. Caso tenham alguma ideia interessante que possamos colocar em pauta para o video, **registrem aqui**
